ansible_collections/arista/avd/molecule/eos_designs_unit_tests/intended/structured_configs/SVI_PROFILE_NODE_2.yml

```diff
@@ -55,18 +55,6 @@ static_routes:
 - vrf: MGMT
   destination_address_prefix: 0.0.0.0/0
   gateway: 10.0.0.1
-- destination_address_prefix: 10.4.10.0/24
```

Comment:
Could you add ip addresses to avoid these ones getting removed. The behavioral change is fine, but should be tested in specific vlans